.circleci/config.yml

```diff
@@ -1,14 +1,27 @@
+#
+# Copyright 2024 Adobe. All rights reserved.
+# This file is licensed to you under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+# OF ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+#
+
 version: 2.1
```

Comment:
We try to make most of the extensions to use the similar workflow. Should we do the same for edge media?
https://github.com/adobe/aepsdk-analytics-android/blob/dev-v3.0.0/.circleci/config.yml
https://github.com/adobe/aepsdk-edge-android/blob/dev-v3.0.0/.circleci/config.yml